firestarter/common/validations.py

```diff
@@ -29,6 +29,7 @@ def helper_get_config_schema(schema_path: str) -> dict:
 
 
 def validate_config(config_path: str, schema_path: str, context = None) -> dict:
+    yaml=YAML()
```

Comment:
Why not type="safe" here like the others 🤔

https://github.com/prefapp/firestarter-workflows/pull/146/files